core/controller/src/main/scala/whisk/core/controller/Triggers.scala

```diff
@@ -143,9 +141,11 @@ trait WhiskTriggersApi extends WhiskCollectionAPI {
             response = ActivationResponse.success(payload orElse Some(JsObject())),
             version = trigger.version,
             duration = None)
+
           logging.info(this, s"[POST] trigger activated, writing activation record to datastore: $triggerActivationId")
-          val saveTriggerActivation = WhiskActivation.put(activationStore, triggerActivation) map { _ =>
-            triggerActivationId
+          WhiskActivation.put(activationStore, triggerActivation) recover {
+            case t =>
+              logging.error(this, s"[POST] storing trigger activation $triggerActivationId failed: ${t.getMessage}")
```

Comment:
How is the user informed of this error?

Reply:
They won’t here. No worse than today. The subsequent improvements will address this deficiency. You can see a sketch of the implementation here https://github.com/apache/incubator-openwhisk/compare/master...rabbah:trigger-activations?expand=1